.circleci/config.yml

```diff
@@ -216,6 +216,26 @@ jobs:
           name: Run danger-swift
           command: danger-swift ci
 
+  "Run Pod spec lint":
+    macos:
+      xcode: "11.2.0"
+    environment:
+      HOMEBREW_NO_AUTO_UPDATE: "1"
+    steps:
+      - checkout
+      - run: 
+          name: Install Cocoapods
+          command: |
+            brew install cocoapods
```

Comment:
The CI has [failed](https://app.circleci.com/jobs/github/broadwaylamb/OpenCombine/379) on this step. Seems like CocoaPods is already installed there. Can you remove this step and see whether it fixes the failure?

Reply:
https://circle-macos-docs.s3.amazonaws.com/image-manifest/v1532/index.html

Exactly, I didn't know that the CircleCI macOS image has CocoaPods 1.8.4 pre-installed. Removing this step should fix the failing CI. :)